content/metadata.yaml

```diff
@@ -70,6 +70,39 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2017-05-26
+  - github: jlevy44
+    name: Joshua J. Levy
+    orcid: 0000-0001-8050-1291
+    email: joshua.j.levy.gr@dartmouth.edu
+    affiliations:
+      - Program in Quantitative Biomedical Sciences, Geisel School of Medicine at Dartmouth, Lebanon, NH
+    funders: Burroughs Wellcome Fund Big Data in the Life Sciences training grant at Dartmouth
+    v1: "Drafted one or more subsections"
+    v2: ""
+    coi:
+      string: "None"
```

Comment:
Can you also add a last approved date? See the other entries. Can make it today.

Reply:
Okay I tried to give it a shot. I wasn't sure if the date was supposed to be when our PRs were merged.

Reply:
The date should be when you last checked the COI information. I think it should be when Brock commented and whenever you last checked it. Can you prod @AlexanderTitus to give a 👍 on inclusion and confirm/approve COI info as correct?